brainscore_language/models/huggingface.py

```diff
@@ -130,16 +130,16 @@ def digest_text(self, text: Union[str, List[str]]) -> Dict[str, DataAssembly]:
     def estimate_reading_times(self, base_output):
         """
         :param base_output: the neural network's output
-        :return: perplexity (calculated as cross entropy) as a proxy for reading times
+        :return: cross entropy as a proxy for reading times, following Smith & Levy 2013
+            (https://www.sciencedirect.com/science/article/pii/S0010027713000413)
         """
         import torch.nn.functional as F
         tokens = self.tokenized_inputs['input_ids'].squeeze()
         logits = base_output.logits.squeeze()
         # assume that reading time is additive,
         # i.e. reading time of multiple tokens is the sum of the perplexity of each individual token
         cross_entropy = F.cross_entropy(logits, tokens, reduction='sum')
```

Comment:
looking at this again, the multiplication by `-1` in the context of reading times is weird since that will give us negative values. Do you typically not worry about reading times being negative? (I know you've been treating this as "proto reading times" that still need to be regressed to actual human reading times, but I would want something that is at least positive to avoid confusion for other users)

Reply:
Oops, I forgot there's already a negative in the cross entropy formula. Yes, surprisal should be positive, so just `cross_entropy / np.log(2)` should be good.